ProjectFiles/Assets/Scripts/Network/Events/ServerPreRoundStartEvent.cs

```diff
@@ -24,7 +24,7 @@ public ServerPreRoundStartEvent(ushort roundNumber, ushort preRoundLength, ushor
             RoundLength = roundLength;
             PlayerCount = playerCount;
             Spaces = spaces;
-            Length = sizeof(ushort) * (Spaces.Count + 5);
+            Length = sizeof(ushort) * (Spaces.Count + 5) + 1;
```

Comment:
This hasn't been done fully yet, but let's try to remove 'magic' numbers

```suggestion
            Length = sizeof(ushort) * (Spaces.Count + 5) + sizeof(byte);
```